src/pages/cart/CartCard.jsx

```diff
@@ -0,0 +1,34 @@
+import "./cart.css";
+import { removeFromCart, updateItemQuantity } from "../../utils";
+import { useProducts } from "../../context/product-context";
+
+
+export function CartCard({value}){
+    const {_id ,image, title, price, originalPrice, qty, discount} = value;
```

Comment:
suggestion -  you can destructure `value` in the argument itself.

Reply:
I tried it but it doesn't work. Will find a solution later.